.github/workflows/pypi.yml

```
@@ -0,0 +1,40 @@
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'main'
+    types:
+      - closed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+      - name: Set up Python
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 19: mapping values are not allowed in this context

(syntax-check)